### PR TITLE
feat: remove LST minting

### PR DESCRIPTION
**Pull request by rafaelugolini**

### CODE REVIEW

File: apps/oeth/src/App.tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ export const App = () => {
       >
         <Stack minWidth={370}>
           <Topnav />
+          <DisableLstBanner />
```
[the remark this answers is not in the document]
Comment (rafaelugolini) [reply apps/oeth/src/App.tsx:28]: @toniocodo didn't test, how can we add an underline to the link? I tried `underline="always"` but it didn't work

[the remark this answers is not in the document]
Comment (toniocodo) [reply apps/oeth/src/App.tsx:28]: underline was removed at the theme level, you can add it back with css
```
<Link sx={{ textDecoration: 'underline' }}>
```
I might update the theme to bring back the underline property

**Comment by toniocodo** [comment]
I updated the theme underlining so that custom css is not overriding the `underline` attribute anymore

**Comment by toniocodo** [comment]
@rafaelugolini @micahalcorn I moved the banner on top and made it sticky as well, lmk wat you think 😃

**Pull request merged**